types/react-big-calendar/index.d.ts

```diff
@@ -148,6 +149,7 @@ export interface Components<TEvent extends Event = Event> {
      * component used as a header for each column in the TimeGridHeader
      */
     header?: React.ComponentType<HeaderProps>;
+    resourceHeader?: React.SFC | React.Component | React.ComponentClass | JSX.Element;
```

Comment:
Would be nice if we could follow the same pattern as for the `header` prop i.e.
```suggestion
    resourceHeader?: React.ComponentType<ResourceHeaderProps>;
```

where `ResourceHeaderProps` matches https://github.com/intljusticemission/react-big-calendar/blob/9332ab72ae84334793b390833df8b32e88a2f1c4/src/ResourceHeader.js#L8-L12

Reply:
Sorry but this is my first contribution and I'm learning how open source world works! Thanks for the remark and I will fix it.

Reply:
No need to apologize. That's what review is for.